### PR TITLE
creator name added

### PR DESCRIPTION
**Pull request by adithyadinesh0412**

### CODE REVIEW

**Comment by adithyadinesh0412** [comment]
Approved on 7th June

File: src/services/resource.js [hunk 1]
```diff
@@ -79,17 +79,36 @@ module.exports = class resourceHelper {
 			}
 
 			const uniqueOrganizationIds = [...new Set(resources.map((item) => item.organization_id))]
+			const uniqueCreatorIds = [...new Set(resources.map((item) => item.user_id))]
+
 			const orgDetailsResponse = await userRequests.listOrganization(uniqueOrganizationIds)
+			const userDetailsResponse = await userRequests.list(
+				common.FILTER_ALL.toLowerCase(),
+				1,
```
Comment (priyanka-TL) [inline src/services/resource.js]: @adithyadinesh0412 what is this 1 ?

File: src/services/resource.js [hunk 2]
```diff
@@ -79,17 +79,36 @@ module.exports = class resourceHelper {
...
+				{ user_ids: uniqueCreatorIds }
+			)
+
 			let orgDetails = {}
+			let userDetails = {}
 			if (orgDetailsResponse.success && orgDetailsResponse.data?.result?.length > 0) {
 				orgDetails = _.keyBy(orgDetailsResponse.data.result, 'id')
 			}
 
+			if (userDetailsResponse.success && userDetailsResponse.data.result.data.length > 0) {
```
Comment (priyanka-TL) [inline src/services/resource.js]: @adithyadinesh0412  make this triggger error if some if the result or data is not there plesae make this like orgDetails condition

File: src/services/resource.js (hunk 1 above)
Comment (priyanka-TL) [reply src/services/resource.js]: @adithyadinesh0412 make it empty instead of 1

**Pull request merged**